etc/config.xml

```diff
@@ -66,6 +66,7 @@
                 <title>Stored Cards (Adyen)</title>
                 <model>AdyenPaymentCardVaultFacade</model>
                 <require_cvc>1</require_cvc>
+                <allow_multistore_tokens>1</allow_multistore_tokens>
```

Comment:
Since the configuration field was defined under `payment/adyen_abstract/allow_multistore_tokens` path, default value should be under `<adyen_abstract>` node of `config.xml`.